### DISCUSSION TITLE
Verification of student status

### DISCUSSION THREAD

**Discussion by sgcoding2016** [community/community#144478] [GitHub Education]
### Hi everyone,

I am trying to set up a Github Education student account. I have a previously set up Github account that is not in my personal name, it uses an alias. I have sent a copy of my student registration and all the required documents and the application has been rejected, which makes no sense. 

Please advise how I can sort out this issue.

**Comment by lllHunterlllSV** [comment]
**Hi there,**

I understand that this situation must be quite frustrating, but don't worry, I'm here to help! When applying for a GitHub Education student account, it's important that your account and the documents you submit align properly. Here are a few steps that might help resolve the issue:

### 1. **Ensure your documents match the name on your GitHub account:**
GitHub Education requires that the name on your student documents matches the name associated with your GitHub account. If your GitHub account is using an alias, this could be the reason why your application was rejected. I would recommend updating your GitHub account to reflect your real name (the one on your student registration documents). You can easily change the name associated with your GitHub account through your account settings.

- [How to update your GitHub profile](https://docs.github.com/en/account-setup-and-management/account-settings-and-configuration/updating-your-profile-information)

### 2. **Make sure your documents are clear and valid:**
Double-check that the documents you’re submitting, such as your student registration or any other required paperwork, are clear, legible, and in the correct format. It's also important that the document clearly shows your enrollment status and the school you're attending. GitHub Education usually accepts documents in PDF or image format (JPEG, PNG).

- [GitHub Education verification process](https://education.github.com/pack)

### 3. **Appeal the decision:**
If you have already submitted the required documents and your application was still rejected, I suggest reaching out to GitHub Education support. In your message, explain the situation clearly and provide any additional information that might help clarify the issue.

- [GitHub Education Support](https://support.github.com/contact)

### 4. **Consider updating your GitHub name temporarily:**
If possible, updating your GitHub account name to match your real name temporarily while applying for GitHub Education may help avoid this issue.

- [How to change your GitHub username](https://docs.github.com/en/account-setup-and-management/account-settings-and-configuration/changing-your-github-username)

### 5. **Check the eligibility and requirements:**
Ensure that you meet the eligibility criteria for GitHub Education, including being a student enrolled at a recognized educational institution. Sometimes, the application can be rejected due to ineligibility, so it's important to review the requirements carefully.

- [GitHub Education eligibility requirements](https://education.github.com/pack)

---

I truly hope this helps, and I wish you the best of luck with your GitHub Education application! If you need any further assistance or have more questions, feel free to reach out anytime. 😊
